examples/headless/Cargo.toml

```diff
@@ -17,3 +17,6 @@ pollster = { workspace = true }
 env_logger = "0.11.2"
 png = "0.17.13"
 futures-intrusive = { workspace = true }
+
+[target.'cfg(target_arch = "wasm32")'.dependencies]
+getrandom = { version = "0.2.12", features = ["js"] }
```

Comment:
Does headless not already depend on `scenes`, seen as you're adding this here

Reply:
Good catch! I added this because `headless` errored out before `scenes`, but yes now that `scenes` has it that's all we need.